mypy/waiter.py

```diff
@@ -37,13 +37,11 @@ def __init__(self, name: str, args: List[str], *, cwd: str = None,
         self.passthrough = passthrough
 
     def start(self) -> None:
-        if self.passthrough is None:
+        if self.passthrough is None or self.passthrough < 0:
             self.outfile = tempfile.TemporaryFile()
         else:
             if self.passthrough >= 0:
```

Comment:
Even without xdist I can't get pytest to not print the row of dots. So agreed, let's keep the existing behavior of runtests.py not passing through any output on -q.